### ISSUE TITLE
enhancement error agreggations

### ISSUE THREAD

**Issue by leandrob04** [Yelp/elastalert#1181]
I was running my rule normally, but  when I updated this problem came up. Can someone help me?
![teste](https://user-images.githubusercontent.com/28788376/27453539-30ce867c-576d-11e7-9702-fd60c7bf5cd9.png)
![teste2](https://user-images.githubusercontent.com/28788376/27453540-32c299f0-576d-11e7-9885-131b4bed18dd.png)

**Comment by Qmando** [comment]
What did you update? This doesn't touch Elastalert code at all. Why don't you add ``print res`` so you can see whats changed.

**Comment by leandrob04** [comment]
Yes, I updated and started the error. I did what you said but the same mistake continues.

**Comment by Qmando** [comment]
The print was for debugging, it wouldn't fix the issue. What did it print out?

**Comment by leandrob04** [comment]
This command works perfectly.
![dwdw](https://user-images.githubusercontent.com/28788376/27501154-f675223a-5841-11e7-8a23-773257ef0115.png)

**Comment by Qmando** [comment]
Paste the output of the print statement you added.
